itests/kit/blockminer.go

```diff
@@ -335,4 +359,8 @@ func (bm *BlockMiner) Stop() {
 	bm.t.Log("shutting down mining")
 	bm.cancel()
 	bm.wg.Wait()
+	if bm.unpause != nil {
+		close(bm.unpause)
+		bm.unpause = nil
```

Comment:
The problem is that we call Stop twice in lots of tests so we get a double close.  This causes a bunch of test faiures in CI.  I think selecting on the nil channel should be ok as long as its done with ctx.Done() in the select as well because we cancel the context before setting nil